_SQLServer/TableLoadHeroes.sql

```diff
@@ -0,0 +1,478 @@
+
+
+--Hexcast Sorceress
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Hexcast Sorceress', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Hexcast_Sorceress_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Hexcast_Sorceress_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/Hexcast_Sorceress.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'Trained in the dark art of hexes and curses the Hexcast Covens are uneasy allies to the rulers of Crystalia.  Nonetheless their potency in battle cannot be denied.  Slowing enemies, shattering armor, and weakening their foes a Hexcast Sorceress can lay even the mightest low.');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Frozen Feel
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Hobbled Hands
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 1); --Broken Bones
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 1); --Wizard Wings
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Human Hero', 6, 3, '2B', '3B', '1B1R', '1B1R', 5, 1);
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,3,1);--! To look at later
+insert into DefenseAssignments (CharacterIndex, AttributeIndex) VALUES (0,2);--! To look at later
+
+
+--Miserable Toad
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Miserable Toad', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Miserable_Toad_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Heroes/Card_Hero_Miserable_Toad_1.0_back.png', 
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Heroes/Miserable_Toad.xhtml', 
+'Hero', '1.0', '1.0 Core Set', 'Default',   
+'She said she wanted my honest opinion.....');
+insert into Characters (CardIndex) VALUES (0);
+insert into KeywordAssignments (CharacterIndex, KeywordIndex) VALUES (1, 0); --! To look at later
+insert into AbilityAssignments (CharacterIndex, AbilityIndex) VALUES (0, 0); --Sticky Tongue
+insert into ExploreCharacters (CharacterIndex, ModelSize, CreatureType, Movement, Actions, Strength, Armor, Will, Dexterity, Health, Potions) VALUES (0, 'Small', 'Shapeshift', 5, 3, '1B', '0', '2B', '3B', 5, 1);--? Should this be small or large - I remember you saying there were only 2 large creatures - this would be 3 ?
+insert into OffenseAssignments (CharacterIndex, AttributeIndex, OffenseRange) VALUES (0,0,1);--! To look at later
+insert into DefenseAssignments (CharacterIndex, AttributeIndex) VALUES (0,4);--! To look at later
```

Comment:
This character should have a 1st defense assignment for armor (0,4)

Reply:
This should be a (0,2), (0,4) already there - per our conversation on 2/27/21.